modeltrans/fields.py

```diff
@@ -109,15 +132,15 @@ def __get__(self, instance, instance_type=None):
         def has_field(field_name):
             return field_name in instance.i18n and instance.i18n[field_name]
 
-        # in two cases, just return the value:
```

Comment:
@jieter hier boven op regel 122-124 lijkt het fout te gaan voor field_i18n, als de current language gelijk staat aan DEFAULT_LANGUAGE, want dan krijg je altijd de DEFAULT_LANGUAGE terug ongeacht of de model instance een andere fallback heeft, want er word niet gecheckt of de DEFAULT_LANGUAGE wel een waarde heeft, dus ook "" word geretourneerd. Toch?